src/wabt.cpp

```diff
@@ -260,14 +333,22 @@ ExecutionResult WabtEngine::execute(
   // FIXME: really bad design
   interface.setWasmMemory(env.GetMemory(0));
 
+  // for rethrowing exception
+  interface.my_exception_ptr = nullptr;
```

Comment:
I don't think this needs to be null'd here.

Reply:
Ok.